Resources/Private/Language/locallang.xlf

```diff
@@ -356,6 +584,18 @@ Date: ###DATE###
 		<trans-unit id="notification.newDocument.submitter.subject">
 			<source>You have registered a document</source>
 		</trans-unit>
+		<trans-unit id="notification.registerDocument.admin.body" xml:space="preserve">
```

Comment:
Same comment as above. Emails texts are usually configured.

Reply:
But default messages have always been configured, this is done in the the same way as before, so it is not mandatory to configure them in the client 